workshop/8-PerformanceOptimization/8.1-Layer0.ipynb

```diff
@@ -187,7 +187,7 @@
     "\n",
     "Please refer to the [Layer0 docs](https://docs.layer0.co) to better understand the possibilities available when working with Layer0. There are additional framework guides available to further test your implementations. \n",
     "\n",
-    "Feel free to reach out to me ronakbsh@amazon.com for questions.\n"
+    "Feel free to reach out to ronakbsh@amazon.com for questions.\n"
```

Comment:
What if you change roles? I would suggest removing this line or changing it to refer readers to Issues on the repo or something.

Reply:
removed it for now.